book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Awesome implementing a method for adding rentals to create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, I think we can be more specific with the arguments passed in as pointed out by the previous reviewer.

<img width="540" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/223720030-7a7db024-6810-4094-b198-a141c6bd2a38.png">

```suggestion
 def add_rental(person, date)
    Rental.new(date, self, person)
    @rentals.push(rental)
  end
```
- [optional] I think you need to indicate the `rental` file path that you want to access in line 1 (`require_relative 'rental'`
)